open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/MapVisualizationType.java

```diff
@@ -73,10 +73,6 @@
     /** The type used for visualizing tiles composed of interpolated data. */
     INTERPOLATED_IMAGE_TILES(MapVisualizationStyleCategory.IMAGE);
 
-    /** Predicate for testing if this is a map data element type. */
-    public static final Predicate<MapVisualizationType> IS_MAP_DATA_ELEMENT_TYPE_PREDICATE = type -> type != null
-            && type.isMapDataElementType();
-
     /** Predicate for testing if this is an annotation layer type. */
     public static final Predicate<MapVisualizationType> IS_ANNOTATION_LAYER_PREDICATE = type -> type == MapVisualizationType.ANNOTATION_POINTS
             || type == MapVisualizationType.ANNOTATION_REGIONS || type == MapVisualizationType.USER_TRACK_ELEMENTS;

```

Comment:
I feel like `MapVisualizationType` is redundant for all of these since this is defined inside the enum itself, they could probably stand to be removed if this is being left in here